src/api/app/models/request_ignoration.rb

```diff
@@ -0,0 +1,17 @@
+class RequestIgnoration < ApplicationRecord
+  belongs_to :staging_workflow
+  belongs_to :bs_request, primary_key: :number
+
+  validates :staging_workflow_id, :bs_request_id, :description, presence: true
+  validates :bs_request_id, numericality: true, uniqueness: { scope: :staging_workflow_id, message: "'%{value}' is already ignored" }
```

Comment:
Use `:bs_request` and not `:bs_request_id` :see_no_evil: 

If you wonder why, the [Rails guide](https://guides.rubyonrails.org/active_record_validations.html#presence) says:

> If you want to be sure that an association is present, you'll need to test whether the associated object itself is present, and not the foreign key used to map the association.

Although it doesn't really tell you why :rofl: The difference is that with `:bs_request` it not only checks the column `bs_request_id` but also makes sure that a BsRequest with that `id` exists. With that `check_bs_request` is not needed anymore

Reply:
I guess you refer to the line 5:
```
validates :staging_workflow_id, :bs_request_id, :description, presence: true
```
So line 5 changed. I left line 6 as it was before.